src/compat/db_admin.c

```diff
@@ -112,6 +112,8 @@ static DB_HOST_STATUS_LIST db_Host_status_list;
 static DB_HOST_STATUS *db_add_host_status (char *hostname, int status);
 static DB_HOST_STATUS *db_find_host_status (char *hostname);
 
+int db_Client_type = DB_CLIENT_TYPE_DEFAULT;
```

Comment:
Is this static? If it is, please also add static keyword to make the storage scope clear.